tests/virt_autotest/install_package.pm

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+use strict;
+use warnings;
+use File::Basename;
+use base "virt_autotest_base";
+use testapi;
+
+sub install_package() {
+    my $self = shift;
+    my $qa_server_repo = get_var('QA_SERVER_REPO', '');
+    if ($qa_server_repo) {
+        type_string "zypper --non-interactive rr server-repo\n";
+        assert_script_run("zypper --non-interactive --no-gpg-check -n ar -f '$qa_server_repo' server-repo");
+    }
+    else {
+        die "There is no qa server repo defined variable QA_SERVER_REPO\n";
+    }
```

Comment:
I think this should be QA_HEAD_REPO actually. That's the variable we use to specify the SP2 URL of QA:HEAD